### PR TITLE
EDX-2690 Add constants for MQTT TLS

### PR DESCRIPTION
**Pull request by FelixTing**
Signed-off-by: Felix Ting <felix@iotechsys.com>

### CODE REVIEW

File: common/tls.go [hunk 1]
```diff
@@ -13,16 +13,29 @@ import (
...
 	OpensslConfFileName = "openssl.conf"
 	RsaKySize           = "4096"
+
+	// Redis specific settings
+	RedisKeyFileName  = "redis.key"
+	RedisCsrFileName  = "redis.csr"
+	RedisCertFileName = "redis.crt"
+
+	// MQTT specific settings
+	MqttServerCertOutputDir = "/tmp/edgex/secrets/ca/mqtt"
```
Comment (cloudxxx8) [inline common/tls.go:31]: why do you need the Dir?  They should be stored in the named volume

File: common/tls.go [hunk 2]
```diff
@@ -13,16 +13,29 @@ import (
...
+	MqttServerKeyFileName   = "server.key"
```
Comment (cloudxxx8) [inline common/tls.go:32]: use a better name.  `server` is too general

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: common/tls.go (hunk 2 above)
Comment (FelixTing) [reply common/tls.go]: This constant is used internally by secretstore-setup service. Removed.

File: common/tls.go (hunk 1 above)
Comment (FelixTing) [reply common/tls.go]: Removed.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**